projects/babi/memnn/memnn_bAbI_10k_task_parameter_sweep.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+
+# Reproduce results from https://arxiv.org/abs/1503.08895 bAbI tasks 1-3 with
+# 10k training examples 
+
+PARLAI=/path/to/ParlAI                 # Where ParlAI exists in filesystem
```

Comment:
is there a way this can default to knowing where it is? we are in the directory after all?